recipe/meta.yaml

```diff
@@ -13,6 +14,9 @@ source:
 
 build:
   number: 100
```

Comment:
Just to clarify what was needed ASAP was NumPy and by extension SciPy. We can continue to discuss `conda` and `conda-build` changes. What we have done here is not pretty, but it let's these packages come into existence in conda-forge without putting undue pressure on anyone else. We fully expect to change this in the long term.